### DISCUSSION TITLE
tpm difference between normal user and sysadmin

### DISCUSSION THREAD

**Discussion by tosim** [TPC-Council/HammerDB#391] [Q&A] [answered]
when i use sa user, the tpm can reach 400,000
but when i use normal use, the tpm is only 250,000

both tests are after build schema successfully.
and the version of hammerDB i use is 2.23.
can anyone help me explain it?

**Comment by tosim** [comment]
test on sqlserver 2017SE

**Comment by sm-shaw** [comment, marked as answer]
Do you see contention on the LOCK_RW_CMED_HASH_SET spinlock? 

This link explains how to diagnose if this is the case https://techcommunity.microsoft.com/t5/datacat/sqlsweet16-episode-8-how-sql-server-2016-cumulative-update-2-cu2/ba-p/305384

If it is having sysadmin privilege can reduce this contention and improve throughput.
